docs/release-notes/wksp-slot-caps-webui.rst

```diff
@@ -0,0 +1,69 @@
+:orphan:
+
+**New Features**
+
+-  WebUI:
+      -  A new section called "Namespace Bindings" has been added to the Create Workspace and Edit
+            Workspace modals. OSS users can input a namespace to which they want to bind their
```

Comment:
since we do not use "OSS users" in our docs we'll just need to point out when a feature is for the enterprise edition. e.g., WebUI: In the enterprise edition, ...

(i.e., all users are OSS users and some users are Enterprise users)